files/materialized_view_for_lanes.sql

```diff
@@ -0,0 +1,113 @@
+create materialized view mv_works_for_lanes
```

Comment:
It's the same except that the join against workgenres is an outer join. This ensures that works that aren't classified under any genres still make it into the materialized view. This is the key insight that lets us use a single materialized view instead of two.

Reply:
Primo 👌